search/templates/search/author_search.html

```diff
@@ -0,0 +1,171 @@
+{%- extends "search/base.html" %}
+
+{% import "search/search-macros.html" as search_macros %}
+
+{% macro select_field(field) %}
+<div class="field is-horizontal">
+  <div class="control">
+    <label class="checkbox">
+      {{ field|safe }}
+      {{ field.label }}
+    </label>
+    {% if field.errors %}
+      <div class="help is-warning">
+        {% for error in field.errors %}
+            {{ error }}
+        {% endfor %}
+      </div>
+    {% endif %}
+  </div>
+</div>
+{% endmacro %}
+
+{% macro advanced_query(form) %}
+
+<div class="columns">
+  <div class="column is-three-fifths">
+    {% if form.errors %}
+    <div class="notification is-warning">
+      Whoops! Something went wrong. Please correct errors in the form below.
+    </div>
+    {% endif %}
+    <div class="box">
+      <form method="GET" action="{{ url_for('ui.author_search') }}">
+
+        {# Author search term fieldset. #}
+        <section data-toggle="fieldset" id="authors-fieldset" class="section">
+          {% for entry in form.authors.entries %}
+            <div class="field has-addons" data-toggle="fieldset-entry">
+              <div class="control is-expanded">
+                <label for="authors-{{loop.index0}}-forename" class="hidden-label">{{ entry.forename.label }}</label>
+                {{ entry.forename(class='input', placeholder="Forename or initial(s)")|safe }}
+                {% if entry.forename.errors %}
+                  <div class="help is-warning">
+                    {% for error in entry.forename.errors %}{{ error }}{% endfor %}
+                  </div>
+                {% endif %}
+              </div>
+              <div class="control is-expanded">
+                <label for="authors-{{loop.index0}}-surname" class="hidden-label">{{ entry.surname.label }}</label>
+                {{ entry.surname(class='input', placeholder="Surname")|safe }}
+                {% if entry.surname.errors %}
+                  <div class="help is-warning">
+                    {% for error in entry.surname.errors %}{{ error }}{% endfor %}
+                  </div>
+                {% endif %}
+              </div>
+              <div class="control">
+                <button type="button"
+                  data-toggle="fieldset-remove-row"
+                  id="authors-{{loop.index0}}-remove"
+                  class="button fieldset-hidden-on-first-row"
+                  aria-label="Remove this search term"
+                  {% if loop.index0 == 0 %}style="visibility:hidden;"{% endif %}
+                  >
+                    {# TODO: need something less hacky to get add/remove buttons the same width. #}
```

Comment:
My suggestion for the less-hacky method: how about I specify a class "is-monospace"? If we use the Lucida Grande font for buttons, the + and - symbols are the same width. We can also apply width-controlling CSS parameters on buttons of a group in this way.

Reply:
That sounds excellent